packages/sdk/src/block/block.js

```diff
@@ -0,0 +1,21 @@
+import {send} from "../send/send.js"
+import {getBlock} from "../build/build-get-block"
+import {decodeResponse as decode} from "../decode/decode.js"
+import {invariant} from "@onflow/util-invariant"
+import {atBlockHeight, atBlockId} from "../sdk"
+
+export function block({sealed = false, id, height} = {}) {
+  invariant(
+    !(id && height),
+    `Cannot pass "id" and "height" simultaneously`
+  )
```

Comment:
Might be good to also have an invariant to prevent passing `sealed` along with an `id` or `height`

Reply:
Added @JeffreyDoyle